<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalizeDialog</class>
 <widget class="QDialog" name="NormalizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Normalise layers by mask</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>800</width>
     <height>522</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="labelMinMask_2">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>50</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Select the layers to be used, associate them to their ecosystem state, and idenfify those that need to be inverted:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeLayers">
        <property name="minimumSize">
         <size>
          <width>540</width>
          <height>220</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>540</width>
          <height>220</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <property name="headerLabels" stdset="0">
         <stringlist>
          <string>Use?</string>
          <string>Layer</string>
          <string>EC State</string>
          <string>Invert?</string>
         </stringlist>
        </property>
        <column>
         <property name="text">
          <string notr="true">Use?</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Layer</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">EC state</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Invert?</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelMinMask">
          <property name="text">
           <string>Mask for the ecosystem type to assess (Min‐value mask):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboMinMask"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelMaxMask">
          <property name="text">
           <string>Mask for the high nature-vale areas (Max‐value mask):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboMaxMask"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelFolder">
          <property name="text">
           <string>Output folder:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="folderLayout">
          <item>
           <widget class="QLineEdit" name="lineFolder"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>Browse…</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelPrefix">
          <property name="text">
           <string>Prefix (optional):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="linePrefix"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelSuffix">
          <property name="text">
           <string>Suffix (optional):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineSuffix"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelClip">
          <property name="text">
           <string>Clip to ecosystem type area (min‐mask):</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="checkClip"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelAdd">
          <property name="text">
           <string>Add outputs to project:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkAdd"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelCSV">
          <property name="text">
           <string>Summary CSV filename:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineCSV"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="textHelp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>520</height>
       </size>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>textHelp</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
